### DISCUSSION TITLE
Only delete photos older than X days

### DISCUSSION THREAD

**Discussion by dojoca** [icloud-photos-downloader/icloud_photos_downloader#659] [Ideas]
I’d love for the script to automatically sync my library, but only delete files older than say 90 days from iCloud.

**Comment by dojoca** [comment]
Bit sad that this still hasn’t been considered :/

**Comment by jhofker** [comment]
I've proposed a change that should do this: https://github.com/icloud-photos-downloader/icloud_photos_downloader/pull/1040

**Comment by dojoca** [comment]
Thanks jhofker, that is awesome :)
